src/components/ImgCarousel.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
```

Comment:
Didn't end up using the image carousel, but still wanted to keep it in case I figure out this problem later.